foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/helper/ConversionManager.java

```diff
@@ -44,6 +48,31 @@
  */
 public class ConversionManager extends CoreConversionManager implements Serializable, Cloneable {
     protected Map defaultNullValues;
+    private static final ZoneId defaultZoneOffset;
+    
+    static {
+        ZoneId tzoneid = null;
+        String tzone = PrivilegedAccessHelper.getSystemProperty("org.eclipse.persistence.conversion.useTimeZone");
```

Comment:
Can we register these properties in `org.eclipse.persistence.config.SystemProperties`?

Reply:
Sure, I don't see a reason why not.